eoscostarica/guias.eoscostarica.io
# update(boilerplate): boilerplate section

### GH Issue

Resolves #220  

### Steps to test
1. Go to boilerplate
1. Check content

#### CheckList
